docs/userguide/forecasting_overview.md

```diff
@@ -196,6 +197,40 @@ pred.plot(label='forecast')
 
 ![TCN quantile regression](./images/probabilistic/example_tcn_quantile.png)
 
+### Capturing model uncertainty using Monte Carlo Dropout
+In Darts, dropout can also be used an additional way to capture model uncertainty, following the approach described in [this paper](https://arxiv.org/abs/1506.02142). This is sometimes referred to as *epistemic uncertainty*, and can be seen as a way to marginalize over a family of models represented by all the different dropout activation functions.
```

Comment:
I recommend writing out the paper's title. It can help with Google searches.

Reply:
Good idea 👍